build_tools_check.js

```diff
@@ -0,0 +1,44 @@
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+let OSName = '';
+if (process.platform === 'win32')   OSName = 'Windows';
+if (process.platform === 'darwin')  OSName = 'MacOS';
+if (process.platform === 'linux')   OSName = 'Linux';
```

Comment:
This is unnecessary. You can use `process.platform` directly.

Reply:
Okay, i just removed that in the latest commit. I wrote it like this because i thought it will make it look more cleaner, but i guess its unnecessary.